### ISSUE TITLE
Unable to deploy new Plex container using Portainer

### ISSUE THREAD

**Issue by chili-bean** [tom472/mediabox#107]
I'm using portainer.  I navigate to containers, then plex.  I press the "duplicate/edit" button, and change the name to "plex2", then press deploy the container, I get a red pop up that states "Failure, Object not found inside the database".  I have tried to duplicate a few other containers, and get the same error.

I'm running docker on Ubuntu 20.04.1 LTS

I ran ./mediabox.sh again, and it mostly is successful with the following message, not sure if this matters:

Final $ should be \$ or $name at -e line 1, within string
syntax error at -e line 1 near "s/ControlPassword=blahblah/ControlPassword=blahblah/g"
execution of e- aborted due to compilation errors.
Final $ should be \$ or $name at -e line 1, within string.
syntax error at -e line 1 near "s/daemonpass/blahblah/g"
Execution of -e aborted due to compilation erros.
Setup Complete - Open a browser and go to... 

I still get the same errors.  As a side note, minio does not want to start so I shut it down.  Not sure if that's related either.

edit:  I found the log files for portainer, it has this entry multiple times:

2020/08/16 16:43:24 http error: Unable to proxy the request via the Docker socket (err=Object not found inside the database) (code=500)

**Comment by Bramblefoot** [comment]
Make sure the original container is off. It could be that the ports are in
use from the original container.
Just an idea...

Thanks,
Bill

"Try not. Do, or do not. There is no try." - Yoda

**Comment by chili-bean** [comment]
Thanks, but I did try that.  There has to be something about the whole compose yaml, since this happens with all the containers.

**Comment by tom472** [comment]
@chili-bean -- I'll try and take a look into this but I'm curious are to what you are actually trying to accomplish with trying to duplicate containers. If I new the purpose we might be able to attack it differently.

For example - if you go to "Stacks" within Portainer you see a warning that "Control" is limited because the stack was created outside of Portainer. The issue you are having might be one of the things that is limited.

For the other errors you are seeing - maybe removing your `.env` file and starting fresh might help.
To do that go into your `mediabox` folder and run `$ rm .env` and then re-run the `mediabox.sh` script.
This will start a fresh `.env` file for you and might clear some of those issues.

--Tom

**Comment by chili-bean** [comment]
Let me preface this by saying while I'm not new to IT, I am new to the concept of docker and lxc containers, and your mediabox seemed like a good place to get started learning about them.  

What I originally wanted to accomplish was to add a new volume to the plex container so I could use a different NFS mount point for my media.  I have since discovered the workaround of mounting my NAS below the existing music et.al. folders and navigating there in plex.

But it was while I was trying to add a volume to plex using portainer that I first encountered the problem.  To troubleshoot, I thought well if I just try to duplicate an existing container with no changes I can at least learn if it was my changes that caused the problem.  It was not, I couldn't get any of the containers to duplicate (I didn't try every one).  I did notice that it complained it wasn't originally created inside portainer.

I will try the .env suggestion.  Feel free to close this issue, as I think a lot of it might be my lack of understanding (or RTFM as with most IT people I've know) and I have a workaround now.

Thanks,

Ken
